internal/aws/ecsutil/client_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package awsecscontainermetrics
+package ecsutil
```

Comment:
the `aws` naming should be kept with all modules committed

```suggestion
package awsescutil
```

Reply:
None of the other aws internal packages follow this suggested convention (are all in an `aws` directory).